### ISSUE TITLE
Default toolchain

### ISSUE THREAD

**Issue by jayvdb** [sagiegurari/cargo-make#743]
### Feature Description
By default the predefined rules in https://github.com/sagiegurari/cargo-make/tree/master/src/lib/descriptor/makefiles use the native `cargo` command.  In order to get them to use a toolchain, I have to write a bunch of extend task like

```toml
[tasks.build]
toolchain = "stable"
```

It would be nice to be able to force a default toolchain for all tasks.

### Describe The Solution You'd Like
Something like:

```toml
[config]
default-toolchain = "stable"
```

**Comment by sagiegurari** [comment]
ya i can take a look at that

**Comment by jayvdb** [comment]
Or perhaps

```toml
[env]
CARGO_MAKE_RUST_CHANNEL = "stable"
```

**Comment by sagiegurari** [comment]
its obviously will be an environment variable and not a new config

**Comment by sagiegurari** [comment]
@jayvdb this is now implemented in the dev branch: 0.36.3
can you please try it out and check if it works well for you?
you will see to set the following env var: CARGO_MAKE_RUST_DEFAULT_TOOLCHAIN to the value you need (default is empty and ignored)

**Comment by jayvdb** [comment]
wfm, thanks!
